testapps/MPDeployServerless/SingleFileNodeFunction/ReadMe.txt

```diff
@@ -0,0 +1 @@
+﻿To be completed - getting started text for the new project.
```

Comment:
?

Reply:
Test file that I have now deleted from the repo. Although I think the VS Toolkit's Node.js wizard created the file so we probably have some stale files in there.